es/lessons/basics/collections.md

```diff
@@ -1,13 +1,13 @@
 ---
-version: 0.9.0
+version: 1.0.0
 layout: page
 title: Colecciones
 category: basics
 order: 2
 lang: es
 ---
 
-Listas, tuplas, listas de palabras clave, mapas, diccionarios y combinadores funcionales
+Listas, tuplas, listas de palabras clave, mapas y combinadores funcionales.
```

Comment:
I have updated this sentence in a recent commit:
https://github.com/doomspork/elixir-school/commit/3afde9ea3aeef3c114fbf7f8af58571925cc38ef#diff-ea6b1eddaf2eb6f0fad19106878c93a8R10

It should read:
> Listas, tuplas, listas de palabras clave y mapas.